extensions/kafka-client/ui/README.asciidoc

```diff
@@ -0,0 +1,22 @@
+= Kafka UI
```

Comment:
I think we should drop that and rather include something in the documentation.

We don't have a similar thing for other extensions and my guess is that it will either be outdated or require duplicated work between the guide and this readme.

Reply:
I reverted the commit with readme